managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+</Message>
+
+#### Adding your existing database as a source
+
+1. From the Airbyte user interface, click **Sources**.
+
+2. Select **Postgres** from the list of sources.
+
+3. Fill in the required information to connect to your existing database:
+    - **Host**
+    - **Port**
+    - **Database Name**
+    - **Username**
+    - **Password**
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+4. Select an SSL Mode and an SSH Tunnel Method.
+
+5. Select **Detect Changes with Xmin System Column** as **Update Method** (other methods are not supported).
+
+4. Click **Set up source**.
+
+#### Adding your new Managed Database for PostgreSQL as a destination
+
+1. Click **Create a connection**.
+
+2. Select **Postgres** from the list of destinations.
+
+3. Fill in the required information to connect to your existing database:
+    - **Host**
+    - **Port**
+    - **Database Name**
+    - **Username**
+    - **Password**
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+4. Select an SSL Mode and an SSH Tunnel Method.
+
+5. Click **Set up destination**.
+
+5. Select the streams (tables) you want to load in your new Managed Database for PostgreSQL, then click **Next**.
+
+6. Edit the remaining options if required, then click **Finish & Sync**.
+
+7. When finished, make sure your data is stored in your new database by connecting to it, and performing a query.
+
+<Message type="important">
+This procedure does not currently work with Managed Databases for PostgreSQL. However, you can import your backup using [Managed Database for PostgreSQL and MySQL](/managed-databases/postgresql-and-mysql/).
+</Message>
+
+### Using Fivetran
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- A Fivetran account and can connect to the [Fivetran console](https://fivetran.com/dashboard)
```

Comment:
```suggestion
- A Fivetran account, and be able to connect to the [Fivetran console](https://fivetran.com/dashboard)
```

Reply:
Or should it be "are able to"? Given that the starting sentence fragment is "You must have..."